panel/panelpluginsmodel.cpp

```diff
@@ -284,7 +284,8 @@ QPointer<Plugin> PanelPluginsModel::loadPlugin(LXQt::PluginInfo const & desktopF
 
 QString PanelPluginsModel::findNewPluginSettingsGroup(const QString &pluginType) const
 {
-    QStringList groups = mPanel->settings()->childGroups();
+    QSettings userSettings(QSettings::UserScope, mPanel->settings());
```

Comment:
I'm afraid this can't be correct. It's just the basic c-tor https://doc.qt.io/qt-6/qsettings.html#QSettings-1.

Reply:
liblxqt converts LXQt::Settings into QSettings with `->settings()`. In this case we modify QSettings scope - and `findNewPluginSettingsGroup` doesn't modify settings.

Reply:
Also please look at https://github.com/lxqt/lxqt-panel/issues/2126#issuecomment-2395190877

Reply:
`LXQt::Settings` *is* a `QSettings`; conversion doesn't make sense. Here, just an empty `QSettings` is created.

Apart from that, `QSettings::UserScope` is the default. It means that user-specific settings are searched *first*, not alone.

Reply:
Oh, somehow it was producing the correct results, maybe because it was in /usr/local. Then probably an absolute path with XDG_CONFIG_HOME is needed. I can close this, or modify it tomorrow.